app/src/main/java/com/example/stardewvalleypockettracker/ChecklistActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.example.stardewvalleypockettracker;
+
+import android.os.Bundle;
+import android.view.View;
+
+public class ChecklistActivity extends BaseActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.toolbar);
```

Comment:
Are we supposed to set the content view to the toolbar layout in each of these activities?  I think these should be the activity specific layouts